### PR TITLE
[Bug]Added disk available capacity monitoring

### PR DESCRIPTION
**Pull request by retime123**
Modify the bug

## Verify this pull request

https://github.com/apache/dolphinscheduler/pull/10030
Through version 3.0.0, E2E test results, found a bug, so fix it  
Therefore, the function added last time is modified again
If yes, go to version 2.0.6

### CODE REVIEW

**Comment by songjianet** [comment]
Please associate the corresponding issue. @retime123

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HeartBeat.java [hunk 1]
```diff
@@ -245,18 +245,18 @@ public static HeartBeat decodeHeartBeat(String heartBeatInfo) {
...
+        heartBeat.maxCpuloadAvg = Double.parseDouble(parts[4]);
+        heartBeat.reservedMemory = Double.parseDouble(parts[5]);
+        heartBeat.startupTime = Long.parseLong(parts[6]);
+        heartBeat.reportTime = Long.parseLong(parts[7]);
+        heartBeat.serverStatus = Integer.parseInt(parts[8]);
+        heartBeat.processId = Integer.parseInt(parts[9]);
+        heartBeat.workerHostWeight = Integer.parseInt(parts[10]);
+        heartBeat.workerExecThreadCount = Integer.parseInt(parts[11]);
+        heartBeat.workerWaitingTaskCount = Integer.parseInt(parts[12]);
+        heartBeat.diskAvailable = Double.parseDouble(parts[13]);
```
Comment (zhongjiajie) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HeartBeat.java]: You just change the position and do nothing on it? Am I right?

Comment (retime123) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HeartBeat.java]: Yes. I didn't think there was a quote anywhere else.
Switch to version 2.0.6 if you can

Comment (zhongjiajie) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HeartBeat.java]: do you mean you want to add disk caption check but when you submit PR and find it already in `dev` branch?

Comment (retime123) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HeartBeat.java]: What I added on the 2.0.6 branch was merged, and then I submitted it to version 3.0 and found a bug! Now I want to fix it

Comment (zhongjiajie) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HeartBeat.java]: In this case you should change your target branch to `dev`

**Comment by zhongjiajie** [comment]
It is a feature instead of bug, remove the milestone

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HeartBeat.java (hunk 1 above)
Comment (retime123) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HeartBeat.java]: I wrote this function on branch 2.0.6, and it also fits this branch!  I have also committed the ** edge of dev, but it is not merged yet

File: script/dolphinscheduler-daemon.sh [hunk 2]
```diff
@@ -33,8 +33,9 @@ echo "Begin $startStop $command......"
 
 BIN_DIR=`dirname $0`
 BIN_DIR=`cd "$BIN_DIR"; pwd`
-DOLPHINSCHEDULER_HOME=`cd "$BIN_DIR/.."; pwd`
+export DOLPHINSCHEDULER_HOME=`cd "$BIN_DIR/.."; pwd`
 
+chmod -R 700 ${DOLPHINSCHEDULER_HOME}/config
```
Comment (kezhenxu94) [inline script/dolphinscheduler-daemon.sh:38]: Hi, why is this change needed?

Comment (retime123) [reply script/dolphinscheduler-daemon.sh:38]: Well, I developed it on the `2.0.6-prepare` branch, this comparison is on the 2.0.5 branch
